### PR TITLE
Update documentation to pass acrolinx tests.

### PR DESCRIPTION
**Pull request by er-1**
Updated URL tags to open a new window when targeting an external website. Also fixed a couple of incorrect URLs to online documentation.

Signed-off-by: Erwan Prioul <erwan@linux.ibm.com>

### CODE REVIEW

File: docs/advtool-install [hunk 1]
```diff
@@ -151,7 +151,9 @@ function myFunction() {
 </div>
 
 <div class="ibm-col-12-9 ibm-padding-top-1 ibm-padding-bottom-1">
-<table class="ibm-data-table ibm-grid" data-widget="datatable" id="distro-table">
+<div id="distro-table"></div>
+
+<table class="bx--data-table">
```
Comment (ThinkOpenly) [inline docs/advtool-install:156]: Could you add this change to the commit message?

File: docs/advtool-updates [hunk 2]
```diff
@@ -157,7 +157,7 @@ function myFunction() {
 
 <div class="ibm-band-fluid ibm-flex ibm-flex--wrap  ibm-padding-bottom-1">
 <div class="ibm-col-12-9">
-<table class="ibm-data-table ibm-grid" data-order="[[1,&quot;dsc&quot;],[0,&quot;dsc&quot;]]" data-ordering="true" data-widget="datatable">
+<table class="bx--data-table" data-order="[[1,&quot;dsc&quot;],[0,&quot;dsc&quot;]]" data-ordering="true" data-widget="datatable">
```
Comment (ThinkOpenly) [inline docs/advtool-updates:160]: same

File: docs/advtool-updates [hunk 3]
```diff
@@ -173,7 +173,7 @@ function myFunction() {
 		</tr>
 		<tr>
 			<td><a href="https://github.com/advancetoolchain/advance-toolchain/releases/tag/at15.0-3" target="_blank">Advance Toolchain 15.0-3</a></td>
-			<td>05/20/2022</td>
+			<td>05/19/2022</td>
```
Comment (ThinkOpenly) [inline docs/advtool-updates:176]: same

File: docs/advtool-updates [hunk 4]
```diff
@@ -189,7 +189,7 @@ function myFunction() {
 		</tr>
 		<tr>
 			<td><a href="https://github.com/advancetoolchain/advance-toolchain/releases/tag/at14.0-4" target="_blank">Advance Toolchain 14.0-4</a></td>
-			<td>02/04/2022</td>
+			<td>02/03/2022</td>
```
Comment (ThinkOpenly) [inline docs/advtool-updates:192]: same

**Review by ThinkOpenly — CHANGES_REQUESTED** [review]
Do you know who "owns" those "ibm.co" links? Maybe they can be updated?

More comments inline...

File: docs/advtool-install (hunk 1 above)
Comment (er-1) [reply docs/advtool-install:156]: Done

File: docs/advtool-updates (hunk 2 above)
Comment (er-1) [reply docs/advtool-updates:160]: Done

File: docs/advtool-updates (hunk 3 above)
Comment (er-1) [reply docs/advtool-updates:176]: Done

File: docs/advtool-updates (hunk 4 above)
Comment (er-1) [reply docs/advtool-updates:192]: Done

**Review by ThinkOpenly — APPROVED** [review]

**Comment by tuliom** [comment]
> That'd be a smarter way to make the change. @tuliom might know who owns the "ibm.co" link.

I don't know who own those ibm.co links
It might be better to stop using them in favor of ibm.biz links.

**Pull request merged**